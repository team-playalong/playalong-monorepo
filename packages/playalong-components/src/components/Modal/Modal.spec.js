import * as React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import PlyModal from './Modal';

describe('PlyModal Component', () => {
  let wrapper;
  const props = {

  };



  beforeEach(() => {
    wrapper = shallow(<PlyModal {...props} />);
  });

  test('Should always be true', () => {
    expect(true).toBe(true);
  });

  test.skip('Snapshot testing', () => {
    const tree = renderer.create(<PlyModal {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
