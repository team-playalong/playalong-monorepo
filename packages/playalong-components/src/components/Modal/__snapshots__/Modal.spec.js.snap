// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlyModal Component Snapshot testing 1`] = `null`;
